.github/workflows/pull_request.yaml

```diff
@@ -39,6 +39,8 @@ jobs:
           ${{ runner.os }}-go-
     - name: Generate Code
       run: |
+        cd $HOME && curl -sSL https://get.helm.sh/helm-v3.2.0-linux-amd64.tar.gz | tar -xzf - linux-amd64/helm && cd -
+        export PATH=$HOME/linux-amd64:$PATH
```

Comment:
Unless I am missing something, this command is not guaranteed to run before the `Testing` step in the `e2e-istio` job that relies on it. 

If that is the case, I think that the idiomatic way of addressing this would be to create a dependency between the two jobs using the `needs` property and use outputs (as documented [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idoutputs), which is what we do with the `kubectl-path` output). The downside of this approach would be that the two jobs would be run sequentially, which seems a bit of an overkill. To get around this we could extract the Helm setup to a separate step and define it on both jobs, seems like a good compromise between cleanliness and efficiency. 

What do you think?

Reply:
So the `codegen` and `testing` steps are currently run in parallel, separately. The `testing` step was already installing helm 3.2, see here: https://github.com/solo-io/wasm/blob/79a4f362fdb1fa9f2ac64df5388674b62b835018/.github/workflows/pull_request.yaml#L77

Which is to say I believe we're actually seeing the same bug here in two different manifestations:

1. In the codegen task, we were never explicitly installing helm at all - so I'm assuming the base image we're using for building here has a version of helm which is now "too new". This is why I'm now explicitly downloading the 3.2 version and adding it to the path for codegen.
2. In the testing task, we were already downloading (the correct version of) helm, but the version on the PATH seems to be the same bad version which doesn't like the `vdev`. This is why I've added the newly downloaded version to the PATH explicitly in the lower job.